modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -143,16 +140,20 @@ public class ItIgniteNodeRestartTest extends IgniteAbstractTest {
             + "  network.nodeFinder.netClusterNodes: {}\n"
             + "}";
 
-    /** Cluster nodes. */
-    private static final List<Ignite> CLUSTER_NODES = new ArrayList<>();
+    @InjectConfiguration
+    private static RaftConfiguration raftConfiguration;
 
-    private static final List<String> CLUSTER_NODES_NAMES = new ArrayList<>();
+    private final List<String> clusterNodesNames = new ArrayList<>();
```

Comment:
Maybe you should use a concurrent collection?

Reply:
why?

Reply:
Can we run a node on another thread? like asynchronously.

Reply:
We can start a node on another thread, but adding a node name to this list always happens in the thread that calls the `startNode` method, so this collection is never accessed concurrently

Reply:
Ok